app/views/early_career_payments/claims/itt_year.html.erb

```diff
@@ -52,20 +50,22 @@
         <% end %>
         <br/>
         <br/>
-        <details class="govuk-details" data-module="govuk-details">
-          <summary class="govuk-details__summary">
-            <span class="govuk-details__summary-text">
-              If you did a part time ITT
-            </span>
-          </summary>
-          <div class="govuk-details__text">
-            <p class="govuk-body">
-              You might still be eligible to claim if your ITT coincided with one of the
-              academic years stated, even if it didn’t start or complete in one of those
-              years.
-            </p>
-          </div>
-        </details>
+        <% if current_claim.eligibility.qualification == "postgraduate_itt" || current_claim.eligibility.qualification == "undergraduate_itt" %>
```

Comment:
To avoid duplication couldn't you do something like `%w[foo bar].include?(something)`?